libs/template/templates/default-sql/template/{{.project_name}}/src/orders_daily.sql.tmpl

```diff
@@ -1,10 +1,9 @@
 -- This query is executed using Databricks Workflows (see resources/{{.project_name}}_sql_job.yml)
-{{- /* We can't use a materialized view here since they don't support 'create or refresh' yet.*/}}
 
 USE CATALOG {{"{{"}}catalog{{"}}"}};
 USE IDENTIFIER({{"{{"}}schema{{"}}"}});
 
-CREATE OR REPLACE VIEW
+CREATE OR REPLACE MATERIALIZED VIEW
```

Comment:
For my information; does this remove incrementalization state?

The SQL ref includes a separate statement to trigger a refresh.

If it does, the template could include a separate job to periodically trigger refreshes on this view.